collectors/state_collector.go

```diff
@@ -45,8 +51,14 @@ func NewStateCollector(lnd *lndclient.LndServices,
 			"time to start in seconds",
 			nil, nil,
 		),
-		startTime: time.Now(),
-		errChan:   errChan,
+		timeToUnlockDesc: prometheus.NewDesc(
+			"lnd_time_to_unlock_secs",
```

Comment:
Rather than seconds, I think we should emit the start/unlock time in a unit of milli-seconds, this way we have more precision, and if we need to go to seconds, we can do so in Prom/Grafana.